src/rospkg/environment.py

```diff
@@ -109,7 +109,7 @@ def get_ros_package_path(env=None):
     """
     if env is None:
         env = os.environ
-    return env.get(ROS_PACKAGE_PATH, None)
+    return env.get(ROS_PACKAGE_PATH, '') + ':/usr/share'
```

Comment:
> Must it? In the proposal, `rospkg` will act like the debian version of `rospack` when the env variable is not set and like the packages.ros.org version of `rospack` when it is. I see no reason to have `rospack` look for the env variable and change behavior like `rospkg`.

I have the same patch for rospack waiting over here...

> While that package can depend on a new version number of rospkg there is no way to prevent users to update only rospkg but not that ros package. The result will be that on Ubuntu the user now has /usr/share in his RPP which will significantly increase the crawling time. This will also be the default case for distributions we have are not building anymore since rospkg will be deployed across all ROS distros.

Actually the second commit limits the crawl, so this is actually almost for
free.

I've renamed the variable as proposed.